scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -426,27 +428,34 @@ class BigQueryClient private (private val projectId: String,
     }
   }
 
-  private[bigquery] def isLegacySql(sqlQuery: String, flattenResults: Boolean): Boolean = {
+  private[scio] def isLegacySql(sqlQuery: String, flattenResults: Boolean): Boolean = {
     def isInvalidQuery(e: GoogleJsonResponseException): Boolean =
       e.getDetails.getErrors.get(0).getReason == "invalidQuery"
     def dryRunQuery(useLegacySql: Boolean): Try[Job] =
       runQuery(sqlQuery, null, flattenResults, useLegacySql, dryRun = true)
 
-    // dry run with SQL syntax first
-    dryRunQuery(false) match {
-      case Success(_) => false
-      case Failure(e: GoogleJsonResponseException) if isInvalidQuery(e) =>
-        // dry run with legacy syntax next
-        dryRunQuery(true) match {
-          case Success(_) =>
-            logger.warn("Legacy syntax is deprecated, use SQL syntax instead. " +
-              "See https://cloud.google.com/bigquery/sql-reference/")
-            logger.warn(s"Legacy query: `$sqlQuery`")
-            true
-          case Failure(f) => throw f
+    sqlQuery.trim.split("\n")(0).trim match {
```

Comment:
Let me fix the case sensitivity here. They indeed are not supposed to be case sensitive per:
https://cloud.google.com/bigquery/docs/reference/standard-sql/enabling-standard-sql